data_collection/web/javascript_tracking_client/api.rst

```diff
@@ -684,6 +805,13 @@ E-commerce
     ``productSKU``, ``productName`` and ``productPrice`` parameters supplying
     ``undefined`` where necessary.
 
+    .. deprecated::
+
+        Data gathered by this function is not used in any predefined ecommerce reports in new Analytics UI.
+        It is still used in legacy UI.
+
+        Please mgrate to the :ref:`e-commerce v2<jtc-api-ecommerce-v2>` function :ref:`ecommerceProductDetailView<jtc-api-ecommerceProductDetailView>`.
```

Comment:
Typo here: `mgrate`.

Reply:
I think we should also remove `It is still used in legacy UI` or at least we should phrase it differently. 
Proposal:
`It is still used in older Piwik PRO versions and is kept for backwards compatibility purposes.`